### PR TITLE
API client for Nodejs client

### LINKED ISSUE #190: Delete Authentications with nodejs client
Hey, thanks for this amazing OpenSource project! It helps me a lot to manage service integration :muscle: 

In my use case, I manage third-party service in my App and store the authId when the User has connected the service. But I would like to remove the Authentification with the nodejs client when the User delete (or disable) the service connection.

For example,

```TypeScript
import Pizzly from 'pizzly-node';

const pizzly = new Pizzly({
      host: 'http://localhost:8080',
      secretKey: 'mySecretKey',
    });

const gDrive = pizzly.integration('google-drive');

// Delete the Authentification object to clean it
gDrive.auth('my-auth-uuid').deleteAuthentification()
```

Thanks in advance for your help/support.

Best!

### PR DESCRIPTION
**Pull request by Nightbr**
Hey guys!

closes #190 

## Proposed changes

I would like to introduce the api part of the Nodejs client.

With the NodeJs client, you can know get access to the API of Pizzly.

```TypeScript
const pizzly = new Pizzly({ host: 'example.org', secretKey: 'myKey' })
const apiClient = pizzly.api('google-drive')

// Manage configuration through API
await apiClient.get('configurations')
await apiClient.put('configurations/72184458-7751-41fe-8dcc-0251ab2cc578', {
  body: { credentials: { clientId: 'abcd***************', clientSecret: '1234***********************' } }
})
await apiClient.delete('configurations/72184458-7751-41fe-8dcc-0251ab2cc578')

// Manage Authentifications through API
await apiClient.get('authentifications')
await apiClient.post('authentifications/1994cc00-a4d6-11ea-9187-b798bad9d2ac/refresh')
await apiClient.delete('authentifications/1994cc00-a4d6-11ea-9187-b798bad9d2ac')
```

I would like to discuss if this approach is ok for you guys, and when we will have an agreement, I will update the documentation according to the decision.

Feel free to ask me anything.

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/bearer/pizzly/blob/master/README.md#contributing-guide) guide
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

Cheers & thanks in advance for your help & review!

### CODE REVIEW

**Comment by markmichon** [comment]
Hey @Nightbr, sorry about the delay on this. I can merge it, but we're sorting out some permission issues on the npm release so it won't go live if merged. I'm hoping within the next week or two we should have it sorted.

**Comment by Nightbr** [comment]
Yeah no problem, it can wait, just tell me what do you think of the approach and I will update the documentation before merging it.

Thanks 👌

**Comment by markmichon** [comment]
@Nightbr quick update. Hoping to get things sorted out this week. As far as the approach, lgtm 👍

**Comment by Nightbr** [comment]
Ok, I will update the doc according to the implementation 👌 thanks

**Comment by Nightbr** [comment]
@markmichon Documentation is ready :ok_hand:  Feel free to review & ask me anything ;)

File: docs/API.md [hunk 1]
```diff
@@ -13,10 +13,13 @@ The main endpoints are:
 - `/api/:integration/authentications/:authId` - To request a specific authentication
 - `/api/:integration/authentications/:authId/refresh` - To refresh a specific authentication
 
-While most actions are available through the dashboard, there are two common use cases wit Pizzly's API:
+While most actions are available through the dashboard, there are two common use cases with Pizzly's API:
 
 - saving an integration's configuration on-the-fly (`POST /api/:integration/configurations`);
 - retrieving an OAuth payload (`GET /api/:integration/authentications`).
+- delete an authentification (`GET /api/:integration/authentications/:id`).
```
Comment (markmichon) [inline docs/API.md:20]: Is this meant to be a DELETE rather than a get?

**Review by markmichon — CHANGES_REQUESTED** [review]

File: docs/API.md (hunk 1 above)
Comment (Nightbr) [reply docs/API.md]: Good catch, pushed a fixed :ok_hand:  Thanks!

**Review by markmichon — APPROVED** [review]
LGTM. I'll leave an additional comment once this is live on NPM

**Comment by markmichon** [comment]
Alright, should be live. Thanks for all your hard work—and patience—on this!

**Pull request merged**